sharepoint/sharepoint-ps/sharepoint-online/Get-SPOListFileVersionExpirationReportJobProgress.md

```diff
@@ -0,0 +1,99 @@
+---
+external help file: sharepointonline.xml
+Module Name: Microsoft.Online.SharePoint.PowerShell
+online version: https://learn.microsoft.com/powershell/module/sharepoint-online/get-spolistfileversionexpirationreportjobprogress
+applicable: SharePoint Online
+title: Get-SPOListFileVersionExpirationReportJobProgress
+schema: 2.0.0
+author: msjennywu
+ms.author: jennywu
+ms.reviewer:
+manager: seanmc
+---
+
+# Get-SPOListFileVersionExpirationReportJobProgress
+
+## SYNOPSIS
+
+Gets the status for a file version usage report generation job for a document libray.
```

Comment:
>usage [](http://example.com/codeflow?start=35&length=5)

should there be "expiration" in this sentence? since the cmdlet has "Expiration"in it

Reply:
Updated it as file version expiration report to be consistent with cmdlet name, also other cmdlets.